### PR TITLE
add search guide and update yml

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+[UPDATE EDGE](../13.edge-statements/2.update-edge.md) |Updates properties on an edge.
+[UPSERT EDGE](../13.edge-statements/3.upsert-edge.md) | Updates or inserts an edge.
+[DELETE EDGE](../13.edge-statements/4.delete-edge.md) | Deletes one or more edges.
+
+## Native statements (TODO)
+
+Statements | Description
+---|---
+
+## Full-text statements (TODO)
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md:46]: Native index statements and Text-based index statements?

File: docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Statements | Description
+---|---
+
+## Subgraph and path statements
+
+Statements | Description
+---|---
+[GET GRAPH](../16.subgraph-and-path/1.get-subgraph.md) | Retrieves subgraphs based on the specified conditions.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md:55]: GET SUBGRAPH

**Review by whitewum — APPROVED** [review]

**Review by oldLady344 — APPROVED** [review]

File: docs-2.0/3.ngql-guide/1.nGQL-overview.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ This document gives an introduction to the query language of Nebula Graph, nGQL
 
 ## What is nGQL
 
-nGQL is a declarative graph query language for Nebula Graph. It allows expressive and efficient graph patterns. nGQL designed for both developers and operations professionals. nGQL is an SQL-like query language, so it's easy to learn. nGQL is a project in progress. New features and optimizations are done steadily. There can be differences between syntax and implementation.
+nGQL is a declarative graph query language for Nebula Graph. It allows expressive and efficient graph patterns. nGQL designed for both developers and operations professionals. nGQL is an SQL-like query language, so it's easy to learn. nGQL is a project in progress. New features and optimizations are done steadily. There can be differences between syntax and implementation. Nebula Graph 2.0 version or above supports [openCypher 9](https://www.opencypher.org/resources).
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/1.nGQL-overview.md:7]: to describe the version, "later" and "earlier" are preferred.

File: docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+| [UPDATE EDGE](../13.edge-statements/2.update-edge.md) | Updates properties on an edge.                |
+| [UPSERT EDGE](../13.edge-statements/3.upsert-edge.md) | Updates or inserts an edge.                   |
+| [DELETE EDGE](../13.edge-statements/4.delete-edge.md) | Deletes one or more edges.                    |
+
+## Native index statements
+
+| Statements                                                                 | Description                                                               |
+| -------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| [CREATE INDEX](../14.native-index-statements/1.create-native-index.md)     | Creates native index for one or more properties of a tag or an edge type. |
+| [SHOW INDEXES](../14.native-index-statements/2.show-native-indexes.md)     | Lists the defined tag or edg type indexes names.                          |
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md:46]: index names

File: docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+| ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------- |
+| [SUBMIT JOB COMPACT](../18.operation-and-maintenance-statements/4.job-statements.md#submit_job_compact) | Triggers the long-term RocksDB compact operation.                     |
+| [SUBMIT JOB FLUSH](../18.operation-and-maintenance-statements/4.job-statements.md#submit_job_flush)     | Writes the RocksDB memfile in memory to the hard disk.                |
+| [SUBMIT JOB STATS](../18.operation-and-maintenance-statements/4.job-statements.md#submit_job_stats)     | Starts a job that makes the statistics of the current graph space.    |
+| [SHOW JOB](../18.operation-and-maintenance-statements/4.job-statements.md#show_job)                     | Shows the information of a specific job and all its tasks.            |
+| [STOP JOB](../18.operation-and-maintenance-statements/4.job-statements.md#stop_job)                     | Stops jobs that are not finished.                                     |
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md:83]: the jobs that ...

File: docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md (hunk 4 above)
Comment (Amber1990Zhang) [reply docs-2.0/3.ngql-guide/2.nav.md/1.search-guide.md:46]: not here. just keywords here

File: docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
+# Deploy full-text index
+
+Nebula Graph full-text indexes are powered by [Elasticsearch](https://en.wikipedia.org/wiki/Elasticsearch).  This means that you can use Elasticsearch full-text query language to retrieve what you want. Full-text indexes are managed through built-in procedures. They can be created only for variable `STRING` and `FIXED_STRING` properties when the listener cluster and the Elasticsearch cluster are deployed.
```
Comment (oldLady344) [inline docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md:3]: This means ==> extra space

File: docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+Make sure that you specify the following fields in strict accordance with the preceding template format:
+
+```json
+"template": "nebula*"
+"tag_id" : { "type" : "long" },
+"column_id" : { "type" : "text" },
+"value" :{ "type" : "keyword"}
+```
+
+You can configure the Elasticsearch to meet your business needs.  To customize the Elasticsearch, see [Elasticserch Document](https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html).
```
Comment (oldLady344) [inline docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md:43]: .  To customize==> extra space

File: docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+## Sign in to the text search clients
+
+```ngql
+SIGN IN TEXT SERVICE [(<elastic_ip:port> [,<username>, <password>]), (<elastic_ip:port>), ...]
+```
+
+When the Elasticsearch cluster is deployed, use the `SIGN IN` statement to sign in to the Elasticsearch clients. Multiple `elastic_ip:port` pairs are separated with commas. You must use the actual IPs and the port number in the configuration file for the Elasticsearch. For example:
```
Comment (oldLady344) [inline docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md:51]: remove "actual"

File: docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/3.deploy-listener.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
+# Deploy Raft Listener for Nebula Storage service
+
+## Before you start
+
+Before you start using the full-text index, please make sure that you know the [restrictions](../../4.deployment-and-installation/6.deploy-text-based-index/1.text-based-index-restrictions.md).
+
+Full-text index data is written to the Elasticsearch cluster asynchronously. To use the full-text index search, you must run one or more nebula-storaged process as the Raft Listener (short for Listener). The listener is a separate process and receives requests from Nebula Graph clients. It manages the traffic of these requests to the database server.
```
Comment (oldLady344) [inline docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/3.deploy-listener.md:7]: nebula-storaged process==> nebula-storaged processes

Comment (oldLady344) [inline docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/3.deploy-listener.md:7]: If "listener" is a proper noun, keeping its capitalization consistent may make it more understandable.

**Review by oldLady344 — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**